### ISSUE TITLE
Add Chinese translation

### ISSUE THREAD

**Issue by amilys** [netbox-community/netbox#14855]
### Language

Chinese

### ISO 639-1 code

zh

### Volunteer

Yes

**Comment by KongNan** [comment]
Please allow us to contribute to the Chinese translation! 中文翻译
https://github.com/netbox-community/netbox/discussions/14815

**Comment by jeffgdotorg** [comment]
Thank you for your interest in helping translate NetBox to Chinese! We're eager to support as many languages as possible, and it's fantastic that you've indicated a willingness to bring your own skills to bear on the effort.

Please be patient as we sort out our a repeatable process for bringing new translation projects online in the third-party translation platform that we use for this work. When we're ready to start work on this translation, we will contact you via this issue with instructions on how to move forward.

**Comment by amilys** [comment]
> Please allow us to contribute to the Chinese translation! 中文翻译 #14815

#14877
![image](https://github.com/netbox-community/netbox/assets/11211624/7f7ef2e1-0984-44f7-a9cf-45dbe53c612e)

我已经提交部分中文翻译，你可以复制`netbox/translations/zh/LC_MESSAGES/django.po`到`netbox/translations/en/LC_MESSAGES/django.po`
然后执行以下命令来使用它
```bash
source /opt/netbox/venv/bin/activate
./manage.py compilemessages
systemctl restart netbox netbox-rq
```

**Comment by abhi1693** [comment]
Could you please rewrite your comments in English?

**Comment by amilys** [comment]
> 你能用英文重写你的评论吗？



> Could you please rewrite your comments in English?

Just telling that friend to temporarily use Chinese translation methods

```bash
# Copy the Chinese translation file to the English translation folder
cp /opt/netbox/translations/zh/LC_MESSAGES/django.po /opt/netbox/translations/en/LC_MESSAGES/django.po

# Activate the Python virtual environment
source /opt/netbox/venv/bin/activate

# Compile messages
./manage.py compilemessages

# Restart NetBox service and NetBox RQ
systemctl restart netbox netbox-rq
```

**Comment by roberthon** [comment]
> > Please allow us to contribute to the Chinese translation! 中文翻译 #14815
> 
> #14877 ![image](https://private-user-images.githubusercontent.com/11211624/298268049-7f7ef2e1-0984-44f7-a9cf-45dbe53c612e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTA4NDQ0MzgsIm5iZiI6MTcxMDg0NDEzOCwicGF0aCI6Ii8xMTIxMTYyNC8yOTgyNjgwNDktN2Y3ZWYyZTEtMDk4NC00NGY3LWE5Y2YtNDVkYmU1M2M2MTJlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMTklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzE5VDEwMjg1OFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTM5MWEzZTVlYWIyZTFjODkxMmE3NmMyMjNjMTcwMjg0OGEyNWM2N2UwOWVjMWFlNjlmZWM1ZmFhMTY4ZDUyMzEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.chAiX1w-t70WGHo5KwrzsFeC-RnQD-48sPFHMbWljbU)
> 
> 我已经提交部分中文翻译，你可以复制`netbox/translations/zh/LC_MESSAGES/django.po`到`netbox/translations/en/LC_MESSAGES/django.po` 然后执行以下命令来使用它
> 
> ```shell
> source /opt/netbox/venv/bin/activate
> ./manage.py compilemessages
> systemctl restart netbox netbox-rq
> ```
How can I use Chinese temporarily in the docker environment, please?
docker运行时怎么临时使用啊大佬

**Comment by Songxwn** [comment]
https://explore.transifex.com/netbox_cn/netbox-cn/

It's better to do it yourself

**Comment by jeffgdotorg** [comment]
> https://explore.transifex.com/netbox_cn/netbox-cn/
> 
> It's better to do it yourself

I content it's better to do it together.

Please consider joining the [netbox-community](https://app.transifex.com/netbox-community/teams/) project. We're a small team and have been very busy with other priorities (especially the 4.0 release), but we're making good progress and now have Transifex auto-sync working.

**Comment by Songxwn** [comment]
https://songxwn.com/NetBox-Docker-cn/


The translation has been completed, you can refer to this tutorial.

**Comment by KongNan** [comment]
> now have Transifex
So please just add Chinese(中文) language to [transifex](https://explore.transifex.com/netbox-community/netbox/) language list :)

**Comment by jeremystretch** [comment]
Support for Chinese will be available in NetBox v4.0.3. Please register on [Transifex](https://explore.transifex.com/netbox-community/netbox/) if you'd like to help improve these translations.
